core/src/main/java/hudson/model/AbstractProject.java

```diff
@@ -105,11 +105,8 @@
 import javax.annotation.CheckForNull;
 import javax.annotation.Nonnull;
 import javax.servlet.ServletException;
-import jenkins.model.BlockedBecauseOfBuildInProgress;
-import jenkins.model.Jenkins;
-import jenkins.model.JenkinsLocationConfiguration;
-import jenkins.model.ParameterizedJobMixIn;
-import jenkins.model.Uptime;
+
+import jenkins.model.*;
```

Comment:
Please do not do it in the production code.
It complicates code reviews and may potentially cause varuous conflicts between class imports